lib/sneakers/errors.rb

```diff
@@ -0,0 +1,5 @@
+require 'timeout'
+
+module Sneakers
+  class JobTimeout < Timeout::Error; end
```

Comment:
> Or even Sneakers::Timeout.

I did try that as well, but 29 specs failed. I see the last one is about
> +"private method `timeout' called for Sneakers::Timeout:Class"

It looks like the stdlib `Timeout` constant gets resolved to `Sneakers::Timeout`. Let me know if you insist on that name, but it will come with an overhead, probably prefixing the `::Timeout` will do, but still not ideal.